docs/developer-docs/defi/tokens/token-standards.mdx

```diff
@@ -0,0 +1,197 @@
+---
+keywords: [intermediate, defi, overview, tokens, icrc-1, icrc-2, icrc-3]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Token standards
+
+<MarkdownChipRow labels={["Intermediate", "Concept"]} />
+
+## Overview
+
+Tokens deployed on ICP must adhere to a standard. The network's native token, ICP, adheres to its own standard as it is unique on the network in the sense that it is used for network utility and governance. Tokens created for other purposes on the network typically adhere to one of the ICRC standards.
+
+[ICRC](https://github.com/dfinity/ICRC) stands for `Internet Computer Request for Comments'. ICRC standards are proposed by the community working group and are not specific to tokens; they can be created for anything on ICP, such as services, canister calls, or authentication methods. However, this documentation will detail the ICRC standards that are specific to tokens.
+
+In the past, the DIP-721 standard was promoted and used by projects on the network for deploying non-fungible tokens. The ICRC-7 standard has since been developed as a newer alternative to the DIP-721 standard.
+
+## ICP tokens
+
+The ICP token is the network's native token used for various utility and governance functions, such as:
+
+- Paying for a canister's resources by being converted into cycles.
+
+- Rewarding node providers for contributing resources to the network.
+
+- Rewarding neurons for staking ICP into the neuron and voting on governance proposals.
+
+The ICP token is similar to Bitcoin in the following ways:
+
+-   Each ICP token is divisible 10^8 times.
+
+-   All transactions are stored in the ledger, starting with the genesis initial state.
+
+-   Tokens are entirely fungible.
+
+-   `AccountIdentifier`s are 32 bytes and are roughly the equivalent of the hash of a public key, optionally together with some additional sub-account specifier.
```

Comment:
`AccountIdentifier` is not used in the rest of text (AFAICS). Can we say here that accounts are identified by a pair (owner, sub-account) ?

Reply:
Revised to "accounts are identified by a pair (owner, sub-account)"